packages/manager/apps/web/client/app/domain/webhosting/order/steps/payment/domain-webhosting-order-steps-payment.controller.js

```diff
@@ -39,6 +39,7 @@ export default class {
   }
 
   preparePayment() {
-    this.stepper.autoPayWithPreferredPaymentMethod = this.autoPayWithPreferredPaymentMethod;
+    this.stepper.autoPayWithPreferredPaymentMethod = !!this
```

Comment:
i don't like the ```!!``` much, what about ```angular.isDefined``` ?

Reply:
Hmm in fact, `this.defaultPaymentMethod` will be either an object if the user select it, or will equal to `false` if user don't want to use the payment method.
So the `!!` is used as a shortcut to test `if( typeof this.defaultPaymentMethod === 'object')`. Where `angular.isDefined` won't be appropriate.